python/paddle/tensor/attribute.py

```diff
@@ -45,6 +46,37 @@ def _real_to_complex_dtype(dtype):
 
 
 def is_complex(x):
+    """Return whether x is a tensor of complex data type(complex64 or complex128).
+
+    Args:
+        x (Tensor): The input tensor.
+
+    Raises:
+        TypeError: if x is not a tensor.
+
+    Returns:
+        bool: The data type of the input is complex data type.
```

Comment:
The data type of the input is complex data type will return True, otherwise False.

Reply:
done